app-center-webapps/src/main/webapp/WEB-INF/conf/app-center/portal/group/platform/professional/navigation.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="ISO-8859-1"?>
+<!--
+This file is part of the Meeds project (https://meeds.io/).
+Copyright (C) 2020 Meeds Association
+contact@meeds.io
+This program is free software; you can redistribute it and/or
+modify it under the terms of the GNU Lesser General Public
+License as published by the Free Software Foundation; either
+version 3 of the License, or (at your option) any later version.
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+Lesser General Public License for more details.
+You should have received a copy of the GNU Lesser General Public License
+along with this program; if not, write to the Free Software Foundation,
+Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+-->
+<node-navigation xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://www.gatein.org/xml/ns/gatein_objects_1_3 http://www.gatein.org/xml/ns/gatein_objects_1_3"
+	xmlns="http://www.gatein.org/xml/ns/gatein_objects_1_3">
+	<priority>1</priority>
+	<page-nodes>
+		<node>
+			<name>appCenterAdminSetup</name>
+      <label>#{administration.appCenterAdminSetup}</label>
+			<page-reference>group::/platform/professional::appCenterAdminSetup</page-reference>
```

Comment:
I'm affraid putting this page inside /platform/professional group nav will prevent other admins (*:/platform/administrators) to access it.

Reply:
updated